go/ngql/query_test.go

```diff
@@ -1114,3 +1115,74 @@ func (suite *QueryGraphQLSuite) TestErrorsInInputType() {
 		"l": types.MakeListType(types.MakeCycleType(0)),
 	}))
 }
+
+func (suite *QueryGraphQLSuite) TestVariables() {
```

Comment:
I wanted to make sure all the right pieces were exposed and that variables use the same kind of conversions.